content/articles/understanding-how-cluster-marker-works/index.md

```diff
@@ -0,0 +1,240 @@
+### Understanding how cluster markers work in web mapping using leaflet.js
+ This is no fact again as we are all aware that BIG DATA is the keyword of the 21st century, and it's shocking to know that only a few people understand that a large part of data is categorized as geospatial. Nowadays nearly all of the gadgets we use are satellite-based systems like GPS. Today, the simplest case of displaying data/marker on a map with an acceptable user experience may be a real challenge especially for the user, they might find it difficult to understand or to analyze data at a particular zoom level. For instance, a platform that has users with hundreds of markers on the map. When you zoom out the map, these markers all overlap and make the map appear messy and crowded. To improve this kind of map, it is professional to use the marker clustering method, and this is the most efficient technique of grouping markers together.
```

Comment:
What do you mean by  `This is no fact again as we are all aware that BIG DATA is the keyword of the 21st century`